src/assets/scripts/main.ts

```diff
@@ -0,0 +1,72 @@
+const total = 0;
+
+const fragment = document.createDocumentFragment();
+document.body.appendChild(fragment);
+
+const header = document.createElement("HEADER");
+header.setAttribute("class", "header");
+fragment.appendChild(header);
+
+const headerFrame = document.createElement("DIV");
+headerFrame.setAttribute("class", "header-wrapper");
+header.appendChild(headerFrame);
+
+const logo = document.createElement("A");
+logo.setAttribute("class", "logo");
+logo.setAttribute("href", "#");
+logo.innerHTML = `<img class="logo__img"src="assets/img/logo.png" alt="logo">`;
+headerFrame.appendChild(logo);
+
+const title = document.createElement("P");
+title.setAttribute("class", "header__p");
+title.innerText = `Grand total: $ ${total}`;
+headerFrame.appendChild(title);
+
+const shopingCartFragment = document.createElement("div");
+shopingCartFragment.setAttribute("class", "shoping-cart");
+shopingCartFragment.innerHTML = `<img class="shoping-cart__img" src="assets/img/shopping-cart.png" alt="shopping-cart"> <div class="shoping-cart__number">0</div>`;
+headerFrame.appendChild(shopingCartFragment)
+
+const main = document.createElement("MAIN");
+main.setAttribute("class", "main");
+fragment.appendChild(main);
+
+const footer = document.createElement("FOOTER");
+footer.setAttribute("class", "footer");
+fragment.appendChild(footer);
+
+const footerFrame = document.createElement("DIV");
+footerFrame.setAttribute("class", "footer-wrapper");
+footer.appendChild(footerFrame);
+
+const usersGithub = document.createElement("DIV");
+usersGithub.setAttribute("class", "users-github");
+footerFrame.appendChild(usersGithub);
```

Comment:
These steps are very similar
Maybe it's better to create some kind of function or manager for creating components?